src/V3Randomize.cpp

```diff
@@ -896,6 +843,17 @@ class CaptureVisitor final : public VNVisitor {
         m_ignore.emplace(memberSelp);
     }
 
+    // Returns true if the first class is a superclass of the second class
+    bool isSuperClassOf(AstClass* const classIsp, AstClass* const classOfp) {
```

Comment:
Can you use AstClass::isClassExtendedFrom?

Reply:
Yes, even better. Somehow I missed it.